h5/h5.ino

```diff
@@ -1898,7 +1916,12 @@ void updateAsyncTimerSettings() {
 }
 
 void taskMoveZ(void *param) {
+  static bool mpgFilterConfiguredZ = false;
   while (emergencyStop == ESTOP_NONE) {
+    if (!mpgFilterConfiguredZ) {
+      pcnt_set_filter_value(PCNT_UNIT_1, MPG_PCNT_FILTER);
+      mpgFilterConfiguredZ = true;
+    }
```

Comment:
Seems like `mpgFilterConfiguredZ` is not needed and pcnt configuration should go to where all other pcnt things are configured